LibOS/shim/src/bookkeep/shim_vma.c

```diff
@@ -412,6 +412,17 @@ __set_vma_comment (struct shim_vma * vma, const char * comment)
     vma->comment[len] = 0;
 }
 
+/*
+ * Add bookkeeping for mmap(). "prev" must be passed into the function as
+ * the immediate precedent vma of the allocating address, or NULL if no
```

Comment:
`allocating` -> `allocated`
plus the same issues as with the `__bkeep_munmap` (see the comment under it)

Reply:
Rewritten.